x-pack/plugins/rollup/public/visualize/editor_config.js

```diff
@@ -0,0 +1,79 @@
+/*
+* Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+* or more contributor license agreements. Licensed under the Elastic License;
+* you may not use this file except in compliance with the Elastic License.
+*/
+
+import { editorConfigProviders } from 'ui/vis/editors/config/editor_config_providers';
+
+// Limit agg params based on rollup capabilities
+editorConfigProviders.register((aggType, indexPattern, aggConfig) => {
+  if(indexPattern.type !== 'rollup') {
+    return {};
+  }
+
+  const aggTypeName = aggConfig.type && aggConfig.type.name;
+
+  // Exclude certain param options for terms:
+  // otherBucket, missingBucket, orderBy, orderAgg
+  if(aggTypeName === 'terms') {
+    return {
+      otherBucket: {
+        hidden: true
+      },
+      missingBucket: {
+        hidden: true
+      },
+      orderBy: {
+        fixedValue: '_key',
+        hidden: true
+      },
+      orderAgg: {
+        hidden: true,
+      }
+    };
+  }
+
+  const rollupAggs = indexPattern.typeMeta && indexPattern.typeMeta.aggs;
+  const field = aggConfig.params && aggConfig.params.field && aggConfig.params.field.name;
+  const fieldAgg = rollupAggs && field && rollupAggs[aggTypeName] && rollupAggs[aggTypeName][field];
+
+  if(!rollupAggs || !field || !fieldAgg) {
+    return {};
+  }
+
+  // Set interval and base interval for histograms based on rollup capabilities
+  if(aggTypeName === 'histogram') {
+    const interval = fieldAgg.interval;
+    return interval ? {
+      intervalBase: {
+        fixedValue: interval
+      },
+      interval: {
+        base: interval,
+        help: `Must be a multiple of rollup configuration interval: ${interval}`
```

Comment:
I seem to get an error even when using a valid multiple, though the error doesn't have an explanation:

![image](https://user-images.githubusercontent.com/1238659/46319121-b86cd180-c58d-11e8-9ab1-2d5a3dcac69f.png)

Reply:
1m is actually a calendar interval, whereas 10m is fixed 🙂

Reply:
Bah, of course! I think we'll need some improved error reporting here in the next release.